src/sidebar/util/test/group-list-item-common-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const proxyquire = require('proxyquire');
+
+describe('sidebar.util.groupListItemCommon', () => {
```

Comment:
I forgot to mention this before, we decided to use slash-separated paths in top-level `describe` blocks going forwards, since that is what import paths use in JS. The Python-style period separators was a weird misstep that some files still have.